client/templates/summons-management/bulk-undeliverable/bulk-undeliverable.njk

```diff
@@ -0,0 +1,118 @@
+{% extends "layouts/default.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "moj/components/banner/macro.njk" import mojBanner %}
+{% from "includes/csrf.njk" import csrfProtection %}
+
+{% block page_title %}{{ serviceName }} - Summons Management - Bulk Undeliverable{% endblock %}
+{% block page_identifier %}Summons Management - Bulk Undeliverable{% endblock %}
+
+{% set currentApp = "Summons replies" %}
+
+{% block beforeContent %}
+
+  {% if bannerMessage %}
+    <div class="govuk-!-margin-top-5">
+      {{ mojBanner({
+        type: "success",
+        text: bannerMessage,
+        iconFallbackText: "Success"
+      }) }}
+    </div>
+  {% endif %}
+
+{% endblock %}
+
+{% block content %}
+
+  {% include "includes/errors.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+
+      <h1 class="govuk-heading-xl">Bulk mark as undeliverable</h1>
+
+      <form id="undeliverableJurorNumberForm">
+
+          <h2 class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--m" for="undeliverableJurorNumber">
+              Enter a juror number to mark as undeliverable
+            </label>
+          </h2>
+
+          <p id="undeliverableJurorNumberError" class="govuk-error-message js-hidden">
+            <span class="govuk-visually-hidden">Error:</span>
+            <span id="undeliverableJurorNumberErrorMessage">The error message</span>
```

Comment:
If there's only one error message we put in here, should we just put it in to start and rely on hiding and unhiding?

Reply:
sure, that would probably make more sense wouldn't it 🙃